### ISSUE TITLE
SDWebImage asking disable bitcode in Xcode 14.2 

### ISSUE THREAD

**Issue by kunalvmore** [SDWebImage/SDWebImage#3525]
### New Issue Checklist

* [Yes] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [Yes ] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [Yes ] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios 
 Platform Version        | e.g. 15.4 (iPhone 11 Pro)
 SDWebImage Version      | e.g. 5.0.0 / 5.13.0
 Integration Method      | e.g. cocoapods
 Xcode Version           | e.g. Xcode 13.3 / Xcode 14.2
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | ------
 Demo project link       | --------

### Issue Description and Steps

I am getting this issue on Xcode 14.2 where SDWebImage is asking me to disable bitcode in Xcode 14.2. But the same is working on Xcode 13.3, As I am moving to Xcode 14.2 and I can't disable bitcode as it will impact other Host Projects. 

Please help with this issue. As I want to enable bitcode SDWebImage.

**Comment by dreampiggy** [comment]
Is this related to SDWebImage ? There are no `explicitly enables bitcode` behavior in any repo configuration for SDWebImage

Check your version or integration method (like CocoaPods issue)
